src/EnergyPlus/InputProcessing/IdfParser.hh

```diff
@@ -67,7 +67,7 @@ public:
 
     json decode(std::string const &idf, json const &schema);
 
-    json decode(std::string const &idf, json const &schema, bool &success);
+    json decode(std::string const &idf, json const &schema, bool &success, bool const convertHVACTemplate = false);
```

Comment:
EnergyPlus built-in file conversion defaults to not convert HVACTemplate objects.